src/main/java/com/commercetools/sync/customers/helpers/CustomerSyncStatistics.java

```diff
@@ -6,6 +6,10 @@
 
 public class CustomerSyncStatistics extends BaseSyncStatistics {
 
+    public CustomerSyncStatistics() {
```

Comment:
Why you reverted it ?

Reply:
IT were broken.

Reply:
Looks like a flaky situation related to API consistency, nothing related to the constructor of customer object:

````
com.commercetools.sync.integration.externalsource.producttypes.ProductTypeWithNestedAttributeSyncIT.setup(ProductTypeWithNestedAttributeSyncIT.java:74)
      Caused by: io.sphere.sdk.client.ErrorResponseException: detailMessage: A duplicate value '"key_3"' exists for field 'key'.
      summary: POST https://api.europe-west1.gcp.commercetools.com/java-sync-target/product-types failed  with response code 400 with X-Correlation-ID `java-sync-target/bfd545fe-c93f-4596-b045-4aebbaa83cf8` on 2020-09-30T09:02:21.653Z
      http response formatted body: {
        "statusCode" : 400,
        "message" : "A duplicate value '"key_3"' exists for field 'key'.",
        "errors" : [ {
          "code" : "DuplicateField",
          "message" : "A duplicate value '"key_3"' exists for field 'key'.",
          "duplicateValue" : "key_3",
          "field" : "key"
        } ]
      }
      http response: io.sphere.sdk.http.HttpResponseImpl@630361b7[statusCode=400,headers={date=[Wed, 30 Sep 2020 09:02:21 GMT], content-length=[226], server=[istio-envoy], x-envoy-upstream-service-time=[10], Alt-Svc=[clear], access-control-allow-headers=[Accept, Authorization, Content-Type, Origin, User-Agent, X-Correlation-ID], x-correlation-id=[java-sync-target/bfd545fe-c93f-4596-b045-4aebbaa83cf8], access-control-allow-methods=[GET, POST, DELETE, OPTIONS], Via=[1.1 google], access-control-allow-origin=[*], access-control-max-age=[299], content-type=[application/json; charset=utf-8], server-timing=[projects;dur=9]},associatedRequest=<null>,textInterpretedBody={"statusCode":400,"message":"A duplicate value '"key_3"' exists for field 'key'.","errors":[{"code":"DuplicateField","message":"A duplicate value '"key_3"' exists for field 'key'.","duplicateValue":"key_3","field":"key"}]}]
      SDK: 1.53.0
      project: java-sync-target
      endpoint: POST /product-types
      Java: 1.8.0_252
````

Reply:
86b533e